README.md

```diff
@@ -100,15 +100,23 @@ using the systemd service below. You should run Nitter behind a reverse proxy
 such as [Nginx](https://github.com/zedeus/nitter/wiki/Nginx) or Apache for
 security reasons.
 
+To run Nitter with docker, you will need to install Redis before you run these commands.
+
 To build and run Nitter in Docker:
 ```bash
 docker build -t nitter:latest .
-docker run -v $(pwd)/nitter.conf:/src/nitter.conf -d -p 8080:8080 nitter:latest
+docker run -v $(pwd)/nitter.conf:/src/nitter.conf -d --network host nitter:latest
```

Comment:
> `--network host`

Using host network is a pretty bad advice. Those who choose to use containers should really know what they are doing, and if they don't, they should learn how to publish a port and how it's different from host networking rather than blindly copy-pasting a "presumably sane" command line from README.

There is no such problem in `docker-compose.yml`, it uses regular port publishing, just as README.md suggested before this change.